src/app/(social)/_components/profile-section.tsx

```
@@ -28,24 +29,28 @@ export default function ProfileSection() {
           className="rounded-full border-2 object-cover object-top"
         />
       </div>
 
-      <div className="font-medium">{team.nickname}</div>
+      <div className="font-medium">{user.nickname}</div>
 
       <div
         id="user-profile-description"
         className="text-xs text-gray-400"
-        dangerouslySetInnerHTML={{ __html: team.html_contents }}
+        dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)